src/pages/MyList/component.js

```diff
@@ -25,35 +25,33 @@ const MyList = ({
         >
           <div className="top-margin">
             <Typography.Title>
+              {listInfo ? listInfo.name : 'List'}
```

Comment:
Instead of using ternary operator in all cases on this page it's better to render nothing before you get response from API

Reply:
In this case, I can not do this because of request for listInfo occurred inside <List /> component.
If I do this than fetchMyList() has never happened.